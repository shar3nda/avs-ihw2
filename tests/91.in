Md50ZsjJ
q/^Abuvq^{FktI_p`k9E*w-DC6Qg:SOES6)T0xH8kt4Vb,bXyB!/O	Xah5J);fKhG%Ti