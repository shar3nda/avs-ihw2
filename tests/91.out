Lowercase: 610, uppercase: 614